tests/scalers/new-relic.test.ts

```diff
@@ -18,10 +18,9 @@ import * as fs from 'fs'
 import * as sh from 'shelljs'
 import * as tmp from 'tmp'
 import test from 'ava'
-import {waitForDeploymentReplicaCount, waitForRollout} from "./helpers";
 
 const newRelicApiKey = process.env['NEWRELIC_API_KEY']
-const newRelicAccountId = '3357103'
+const newRelicAccountId = process.env['NEWRELiC_ACCOUNT_ID']
```

Comment:
NEWRELiC_ACCOUNT_ID (you have a lowercase I in the Newrelic variable name might want to ensure that its consistent?

Reply:
Fixed, thanks for the catch